corpysprofiling/corpysprofiling.py

```diff
@@ -114,12 +117,46 @@ def corpus_viz(corpus, stats):
     Examples
     --------
     >>> from corpysprofiling import corpysprofiling
-    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?", 'words')
-    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?", 'topic')
+    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?")
+    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?")['word cloud']
+    >>> plt.figure()
+    >>> plt.imshow(wordcloud, interpolation="bilinear")
+    >>> plt.axis("off")
+    >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?")['df used for bar']
+    >>> df.plot.bar(rot=0, x='words')
+    >>> plt.xticks(rotation=90)
+    >>> plt.xlabel("Words")
     >>> corpysprofiling.corpus_viz("How many species of animals are there in Russia?", 15)
     TypeError: Input must be a string
     """
-    return
+    # Step 1. To get a word cloud
+    wordcloud = WordCloud().generate(corpus.lower())
+    plt.figure()
+    plt.imshow(wordcloud, interpolation="bilinear")
+    plt.axis("off")
+    
+    # Step 2. To get a bar chart to visualize the words and the length of words
+    # To get a list of words from the input text
+    clean_corpus = clean_tokens(corpus)
+    # To get a data frame summary of the words and length of the words
+    df = pd.DataFrame({'corpus': clean_corpus})
+    df = pd.DataFrame(df['corpus'].value_counts())
+    df.reset_index(level=0, inplace=True)
+    df = df.rename(columns={'index': 'words', 'corpus': 'length'})
+    # To limit the number of words to display in the plot
+    if len(df) < 30:
+        df = df
+    else: df = df.head(30)
+    # To make a bar chart
+    df.plot.bar(rot=0, x='words')
```

Comment:
I think this might work better for plotting the bar chart
```
    bar_fig = plt.figure()
    plt.bar(x=df['words'], height = df['length'])
    plt.xticks(rotation=90)
    plt.xlabel("Words")
```